Automattic/simplenote-android
# Fix line height squashing in editor

To fix #450 I had to remove the `lineSpacingExtra` value in the EditText. This unfortunately makes the line heights a bit more compact in the editor now:

<img width="327" alt="screen shot 2017-11-29 at 9 41 30 am" src="https://user-images.githubusercontent.com/789137/33390020-88f4b868-d4e9-11e7-855f-e1659662be48.png">

I also tried using `lineSpacingMultiplier` instead but got the same results.

**To Test**
Add a note with multiple lines, like:

`Line 1
Line 2
Line 3
Line 4`

Using the soft keyboard, delete the `3` in `Line 3`. The line heights of all text should remain the same.

This PR depends on #475
